lib/routes/charge/items/add_item_page.dart

```diff
@@ -263,6 +275,7 @@ class CreateItemPageState extends State<CreateItemPage> {
                   name: _nameController.text.trimRight(),
                   currency: _currencySymbol(),
                   price: double.parse(_formattedPrice()),
+                  sku: _skuController.text,
```

Comment:
Shouldn't it be null in case empty?

Reply:
```suggestion
               sku:  _skuController.text.isNotEmpty ? _skuController.text : null,
```
What are the differences between
`sku: null `versus `sku: ""` ?

Reply:
The difference is "no value" vs "empty string value".
At the repository level  (SQL for example) it is important to have those as null values.

Reply:
[Done](https://github.com/breez/breezmobile/pull/296/commits/3c5761d4edc0927fba540c2fb49ba332fdd6ade2)